src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Sponsors.cshtml

```diff
@@ -1,41 +1,43 @@
-﻿<div class="donate_page">
+@using static DevAdventCalendarCompetition.Resources.ViewsMessages
+
+<div class="donate_page">
     @*<section class="promot_section">
-        <div class="container">
-            <div class="row">
-                <div class="col-md-6 col-md-offset-3">
-                    <div class="promot_content_wrapper">
-                        <div class="promt_content area_title">
-                            <h3>Sponsorzy</h3>
-                        </div>
-                        <div class="santa_with_bag reveal animated" data-reveal-anim="bounceIn" data-anim-delay="0.3s">
-                            <img src="~/theme-2019/images/gift_bag.png" alt="Begger_santa">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-6 col-md-offset-3">
+                        <div class="promot_content_wrapper">
+                            <div class="promt_content area_title">
+                                <h3>Sponsorzy</h3>
+                            </div>
+                            <div class="santa_with_bag reveal animated" data-reveal-anim="bounceIn" data-anim-delay="0.3s">
+                                <img src="~/theme-2019/images/gift_bag.png" alt="Begger_santa">
+                            </div>
                         </div>
                     </div>
                 </div>
             </div>
-        </div>
-    </section>*@
+        </section>*@
 
     @*<div class="container mb-100 mt-100">
-        <div class="row">
-            <div class="col-12 mb-30 mt-30 text-center">
-                <div class="area_title mb-30">
-                    <h3>Chcesz zostać sponsorem i wesprzeć dobrą inicjatywę w 2019 roku? </h3>
-                </div>
+            <div class="row">
+                <div class="col-12 mb-30 mt-30 text-center">
+                    <div class="area_title mb-30">
+                        <h3>Chcesz zostać sponsorem i wesprzeć dobrą inicjatywę w 2019 roku? </h3>
```

Comment:
please add resource here

Reply:
this code was commented out (@* code  *@) do you want to translate?

Reply:
yes :)

Reply:
ok :)